core/opendaq/opendaq/src/instance_impl.cpp

```diff
@@ -659,6 +659,21 @@ ErrCode InstanceImpl::getOnPropertyValueRead(IString* propertyName, IEvent** eve
     return rootDevice->getOnPropertyValueRead(propertyName, event);
 }
 
+ErrCode InstanceImpl::beginUpdate()
+{
+    return rootDevice->beginUpdate();
+}
+
+ErrCode InstanceImpl::endUpdate()
+{
+    return rootDevice->endUpdate();
+}
+
+ErrCode InstanceImpl::getOnEndUpdate(IEvent** event)
+{
+    return rootDevice->endUpdate();
+}
```

Comment:
I feel like this might be misunderstood to mean that the entire instance and its subtree will enter the "isUpdating" state, while this only applies to a single property object.

On that note - would it make sense for nested property objects to also be put into the same state, and its changes applied when the owner stops updating?

Reply:
I would do this at next iteration.